src/librustc_lint/unused.rs

```diff
@@ -395,6 +395,12 @@ trait UnusedDelimLint {
         left_pos: Option<BytePos>,
         right_pos: Option<BytePos>,
     ) {
+        // FIXME(flip1995): Quick and dirty fix for #70814. This should be fixed in rustdoc
```

Comment:
is this actually required? I would expect the check in `emit_unused_delims` to be enough

Reply:
Oh yeah, you're right. I will change it.